fluentd/Dockerfile.centos7

```diff
@@ -58,6 +58,7 @@ RUN mkdir -p /etc/fluent/configs.d/{dynamic,user} && \
     chmod 777 /etc/fluent/configs.d/dynamic && \
     ln -s /etc/fluent/configs.d/user/fluent.conf /etc/fluent/fluent.conf
 
+COPY utils/** ${HOME}/bin
```

Comment:
I think this is wiping out fluentd - it seems to overwrite or remove ${HOME}/bin/fluentd
```
run.sh: line 348: exec: fluentd: not found
```